src/components/AddComment.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 function AddComment() {
+  const [text, setText] = useState('')
+  const [characters] = useState(250)
```

Comment:
make it as a constant `MAX_INPUT_LENGTH` (or kind of) and place out of the component